README.md

```diff
@@ -1 +1,17 @@
-# codacy-test
\ No newline at end of file
+# codacy-test
+
+### Among Codacy’s features
+- Identify new Static Analysis issues
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://release.k8s.dev.codacy.org/gh/DevK8scodacytestuser/codacy-test/pullRequest?prid=3215)
```suggestion

- Identify new Static Analysis issues
```